src/main/java/com/conveyal/r5/profile/PerTargetPropagater.java

```diff
@@ -128,18 +130,20 @@ public PerTargetPropagater(
         // If we're making a static site we'll break travel times down into components and make paths.
         // This expects the pathsToStopsForIteration and pathWriter fields to be set separately by the caller.
         this.calculateComponents = task.makeTauiSite;
+        this.timer = new PropagationTimer();
```

Comment:
In Java this would be entirely convention, and I don't think we have an established convention. In constructors, if a few items need `this` then for readability and uniformity I'd prefer to have `this` in all assignments. I think I got this habit from my early use of Python, where `self` is always needed for instance variables. In an IDE it's not super necessary since syntax highlighting will warn you if you're overwriting a local or static variable, but I still find it clearer to use a lot of `this`. I will update this constructor and other places you commented for uniformity.